Automattic/wp-calypso
# Gutenboarding: Remove disabled vertical input

#### Changes proposed in this Pull Request

This is a follow-up cleanup to #44983 that disabled the vertical input. ("Vertical" in this context means the industry vertical a site might belong to https://public-api.wordpress.com/wpcom/v2/verticals/  ) The `gutenboarding/show-vertical-input` feature flag has been disabled since, so this PR is cleaning up code that's hidden behind it.

Note: We're leaving the verticals data store in place - should it be removed too?

#### Testing instructions

* Verify the removed code is not in use.
* Verify Calypso builds well and tests pass.
* Start on `/new` and verify you don't see a vertical input.
